package.json

```diff
@@ -64,13 +64,13 @@
     "yauzl": "2.10.0"
   },
   "devDependencies": {
-    "@electron-forge/cli": "6.0.0-beta.54",
-    "@electron-forge/maker-deb": "6.0.0-beta.54",
-    "@electron-forge/maker-rpm": "6.0.0-beta.54",
-    "@electron-forge/maker-squirrel": "6.0.0-beta.54",
-    "@electron-forge/maker-zip": "6.0.0-beta.54",
-    "@electron-forge/plugin-auto-unpack-natives": "^6.0.0-beta.61",
-    "@electron-forge/publisher-github": "6.0.0-beta.54",
+    "@electron-forge/cli": "6.0.0",
```

Comment:
Can we update to 6.2.1 or whatever the latest version is actually?

Reply:
Got you